tests/yaml/issue-400.yaml

```diff
@@ -11,6 +11,57 @@ table: |
   nocontractsign 56
   numericnocontchars abcdefghij
 
+# First check if forward translation reflects expected backward translation
+# possibly to be removed or compressed when issue has been fixed.
+tests:
+  # Characters is in numericnocontchars require nocontractsign to cancel numeric mode
+  - - "123abc"
+    - "#abc;abc"
+  # "k" is not in numericnocontchars so does not require contractsign to cancel numeric mode
+  - - "123kabc"
+    - "#abckabc"
+  # whitespace cancels numeric mode
+  - - "123 abc"
+    - "#abc abc"
+  # capsletter cancels numeric mode
+  - - "123Abc"
+    - "#abc,abc"
+  - - "123Kabc"
+    - "#abc,kabc"
+
+  # FIXME: the following tests are only relevant if capsletter fails to cancel numeric mode
+  # once stuck, numeric mode may not be canceled even by nocontractsign
+  - - "123Abckabc" # FIXME: undefined behavior?
+    - "#abc,abck;abc"
+    - {xfail: true}
+    # Actual: '#abc,abckabc'
+    # buevest:
+    # Actual result is correct.
+    # both capsletter and "k" in the output  cancel numeric mode for good.
+    # No need for nocontractsign AFAICS.
```

Comment:
The question is what "#abc,abck;abc" should back translate to. It's clear that the nocontractsign (";") is not needed, but what if it is there? You could argue that the behavior is undefined in this case because when it occurs numeric mode is not active. Or you can argue that the indicator was inserted by accident and should be ignored.